### PR TITLE
MOVE-425: Due to lookup error, using static locator for SMP

### PR DESCRIPTION
**Pull request by martinwam**

### CODE REVIEW

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaSystestConfig.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public TransportProfile getTransportProfile() {
     public LookupClient getElmaLookupClient() throws PeppolLoadingException {
         Mode mode = Mode.of(Mode.TEST);
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaSystestConfig.java]: MAJOR Remove this unused "mode" local variable. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "mode". [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaDevConfig.java [hunk 2]
```diff
@@ -26,7 +26,7 @@ public TransportProfile getTransportProfile() {
     public LookupClient getElmaLookupClient() throws PeppolLoadingException {
         Mode mode = Mode.of(Mode.TEST);
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaDevConfig.java]: MAJOR Remove this unused "mode" local variable. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "mode". [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaStagingConfig.java [hunk 3]
```diff
@@ -29,7 +29,7 @@ public TransportProfile getTransportProfile() {
     public LookupClient getElmaLookupClient() throws PeppolLoadingException {
         Mode mode = Mode.of(Mode.TEST);
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaStagingConfig.java]: MAJOR Remove this unused "mode" local variable. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "mode". [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaProdConfig.java [hunk 4]
```diff
@@ -26,7 +26,7 @@ public TransportProfile getTransportProfile() {
     public LookupClient getElmaLookupClient() throws PeppolLoadingException {
         Mode mode = Mode.of(Mode.TEST);
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ElmaProdConfig.java]: MAJOR Remove this unused "mode" local variable. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "mode". [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)

**Comment by nuth** [comment]
SonarQube analysis reported 8 issues
* MAJOR 8 major

Watch the comments in this conversation to review them.

**Comment by martinwam** [comment]
build

**Pull request closed without merging**